test/mpfr.jl

```diff
@@ -606,7 +606,11 @@ end
         @test log(x) == log(42)
         @test isinf(log(BigFloat(0)))
         @test_throws DomainError log(BigFloat(-1))
-        @test log2(x) == log2(42)
+        if Sys.iswindows()
+            @test_broken log2(x) == log2(42)
+        else
+            @test log2(x) == log2(42)
+        end
```

Comment:
Since I don't think this is necessarily related to the OS, I think it would be better to not make any assumptions about whether this will pass/fail:
```suggestion
        # issue #41450
        @test_skip log2(x) == log2(42)
```